### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by gordonmessmer**
Some of these changes are temporary.  I'd like to merge them in and close a few issues for one more release before removing support for EOL python versions, at which point we can push a new major version.

### CODE REVIEW

**Comment by gordonmessmer** [comment]
@decathorpe Can you look at the PR I just opened for issue #223 ?

File: tests/test_metadata_extractors.py [hunk 1]
```diff
@@ -311,7 +311,7 @@ def setup_method(self, method):
                            ['BuildRequires', 'python-setuptools'],
                            ['BuildRequires', 'python-sphinx']]),
         (1, 'py_modules', ['pytest']),
-        (1, 'packages', ['_pytest']),
+        # (1, 'packages', ['_pytest', 'pytest']), # py34 will get setuptools 43, and packages will not have 'pytest'
```
Comment (decathorpe) [inline tests/test_metadata_extractors.py]: Can you conditionally enable/disable this test depending on the python version?

**Comment by decathorpe** [comment]
Other than the inline comment, looks fine to me ... though I'm wondering if it's actually still necessary to test python2 code.

File: tests/test_metadata_extractors.py (hunk 1 above)
Comment (gordonmessmer) [reply tests/test_metadata_extractors.py]: I've pushed a change to do that.

**Comment by gordonmessmer** [comment]
I'd like to continue testing py27 until issue #182 is resolved, for the sake of platforms that still include and support it.  Once we publish at least one version with support for =~, we can remove support for EOL python versions and rev the major.

**Comment by decathorpe** [comment]
Looks good to me now. Thanks

**Pull request merged**